models/execute.go

```diff
@@ -5,6 +5,10 @@ import (
 	"strings"
 )
 
+type ExecuteRequest struct {
+	QueryParameters map[string]string `json:"query_parameters,omitempty"`
```

Comment:
`map[string]any`

should be any, I don't know of a better way of saying:
it is either string or a float64 or an integer, maybe @diegoximenes  or @jmsgu know..

Reply:
For (int | string) this can be useful: https://pkg.go.dev/k8s.io/apimachinery/pkg/util/intstr#IntOrString.
I am not sure if with the generics support from go 1.18 something nice appeared related to this kind of issue, I haven't used generics on golang yet.
Regarding float, usually I discourage using it for types on API interfaces, due to round trip issues.
Check a little about k8s APIs handling floats with Quantity (https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity) by the way: kubernetes-sigs/controller-tools#245
On Dune's case, maybe if those QueryParameters are used as cache keys somewhere (not sure if they are already being used), then allowing floats can be an issue, but I don't have too much context on Dune yet to know if this is actually a possible scenario hehe.

Reply:
We have to support float, since just restricting parameters to int is not really desirable. I'll make it `any` for now and we can see about using maybe only string in the future and doing a conversion on the duneapi side, or even QES